DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CourseSetup/ManageCourseControllerTests.cs

```diff
@@ -49,5 +49,51 @@ public void Index_returns_ManageCourse_page_when_appropriate_course_found()
             // Then
             result.Should().BeViewResult().WithDefaultViewName().ModelAs<ManageCourseViewModel>();
         }
+
+        [Test]
+        public void Edit_Course_Options_with_course_details_page_opens()
+        {
+            // Given
+            A.CallTo(() => courseDataService.GetCourseDetailsForAdminCategoryId(A<int>._, A<int>._, A<int>._))
+                .Returns(new CourseDetails());
+
+            // When
+            var result = controller.EditCourseOptions(1);
+
+            // Then
+            result.Should().BeViewResult().WithDefaultViewName().ModelAs<EditCourseOptionsViewModel>();
+        }
+
+        [Test]
+        public void Edit_Course_Options_page_when_course_details_are_not_updated()
+        {
+            // Given
+            var courseDetails = new CourseDetails();
+            var editCourseOptionsViewModel = new EditCourseOptionsViewModel(courseDetails);
+            A.CallTo(() => courseDataService.UpdateCourseOptions(A<CourseDetails>._))
+                .Returns(false);
+
+            // When
+            var result = controller.EditCourseOptions(editCourseOptionsViewModel);
+
+            // Then
+            result.Should().BeNotFoundResult();
+        }
+
+        [Test]
+        public void Edit_Course_Options_page_when_coursed_details_are_successfully_updated()

```

Comment:
Same typo on this line. The perils of copy/paste.

Reply:
Sorry again, I have updated the code and is now available for review